### PR TITLE
docs: add how to manage and delete repos

### PR DESCRIPTION
**Pull request by bashbunni**
## What Changed?
added information for 
- [x] renaming repos
- [x] deleting repos
- [x] finding your .repos folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,18 @@ make sure you have added your key as an admin user, or you’re using `anon-acce
 admin-access` in the configuration.
 
 ### A note about RSA keys
```
Comment (aymanbagabas) [inline README.md:255]: I think this line should be removed

File: README.md [hunk 2]
```diff
@@ -253,6 +253,18 @@ make sure you have added your key as an admin user, or you’re using `anon-acce
...
+## Managing Repos
```
Comment (aymanbagabas) [inline README.md:256]: Add new lines after headings

File: README.md [hunk 3]
```diff
@@ -253,6 +253,18 @@ make sure you have added your key as an admin user, or you’re using `anon-acce
...
+`.repos` and `.ssh` directories are created when you first run `soft` at the paths specified for the `SOFT_SERVE_KEY_PATH` and `SOFT_SERVE_REPO_PATH` environment variables. 
+It's recommended to have a dedicated directory for your soft-serve repos and config.
+
+### Deleting a Repo
```
Comment (aymanbagabas) [inline README.md:260]: ditto

File: README.md [hunk 4]
```diff
@@ -253,6 +253,18 @@ make sure you have added your key as an admin user, or you’re using `anon-acce
...
+To delete a repo from your soft serve server, you'll have to remove the repo from the .repos directory.
+
+### Renaming a Repo
```
Comment (aymanbagabas) [inline README.md:263]: ditto

**Review by aymanbagabas — CHANGES_REQUESTED** [review]

**Comment by bashbunni** [comment]
Made the changes requested above!

File: README.md (hunk 1 above)
Comment (aymanbagabas) [reply README.md:251]: Not this one 😆  I meant `### A note about RSA keys`, I should've been more explicit. And while you're at it, could you squash all the commits into one commit?

Comment (bashbunni) [reply README.md:251]: Ah lol! My bad I didn't read that properly haha. And yeah can do!

Comment (bashbunni) [reply README.md]: Done!

**Review by aymanbagabas — APPROVED** [review]

**Pull request merged**